eq-author-api/schema/tests/questionnaire.test.js

```diff
@@ -111,12 +106,6 @@ describe("questionnaire", () => {
       //@todo - doesn't work as expect doesn't always resolve correctly
     });
 
-    it("should resolve section", () => {
```

Comment:
Why have we removed this?

Reply:
Ah yeah these were failing due to the delete section so I took them out meaning to re-write them, must've forgotten. 😅 I'll put something back in now.